pyqtgraph/Qt/__init__.py

```diff
@@ -73,7 +72,7 @@ def write(self, data):
         self.data.append(data)
         
     def getvalue(self):
-        return ''.join(map(asUnicode, self.data)).encode('utf8')
+        return ''.join(map(str, self.data)).encode('utf8')
```

Comment:
we can remove the `.encode('utf8')` bit, as that's default behavior.

Reply:
is it? encode converts it from str to bytes

Reply:
oh, right; yeah disregard me here!

```
$ python
Python 3.9.6 (default, Jul  1 2021, 09:28:20)
[Clang 12.0.5 (clang-1205.0.22.11)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> test = "123"
>>> test.encode("utf8")
b'123'
```